data/json/items/melee.json

```diff
@@ -2082,7 +2082,7 @@
     "looks_like": "pike",
     "techniques": [ "IMPALE", "WBLOCK_2" ],
     "qualities": [ [ "COOK", 1 ] ],
-    "flags": [ "SPEAR", "REACH_ATTACK", "REACH3", "NONCONDUCTIVE" ]
+    "flags": [ "SPEAR", "REACH_ATTACK", "REACH3", "NONCONDUCTIVE", "FRAGILE_MELEE" ]
```

Comment:
Much like the axe, I'm sceptical that even cheap fake piles would be fragile enough to merit the flag.